src/components/Article.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Article({ title, date, text }) {
+  return (
+    <>
```

Comment:
No real need to use fragment here, better to add some wrapper tag - `div` or `article`.

Reply:
Fixed, please have a look